crates/kornia-io/src/functional.rs

```diff
@@ -97,13 +97,13 @@ pub fn write_image_jpegturbo_rgb8(
 /// use kornia_image::Image;
 /// use kornia_io::functional as F;
 ///
-/// let image: Image<u8, 3> = F::read_image_any_rgb8("../../tests/data/dog.jpeg").unwrap();
+/// let image: Image<u8, 3> = F::read_image_any("../../tests/data/dog.jpeg").unwrap();
 ///
 /// assert_eq!(image.cols(), 258);
 /// assert_eq!(image.rows(), 195);
 /// assert_eq!(image.num_channels(), 3);
 /// ```
-pub fn read_image_any_rgb8(file_path: impl AsRef<Path>) -> Result<Image<u8, 3>, IoError> {
+pub fn read_image_any<T: PixelType>(file_path: impl AsRef<Path>) -> Result<Image<T, 3>, IoError> {
```

Comment:
Don’t change this api for now, create a new if required. However, can you sketch here what’s the idea behind dynamic image ? The idea with this effort is to get rid of image crate at all